firstvoices/backend/search/utils/query_builder_utils.py

```diff
@@ -278,3 +278,18 @@ def get_valid_visibility(input_visibility_str):
     if len(selected_values) == 0:
         return None
     return selected_values
+
+
+def get_valid_sort(input_sort_by_str):
+    input_string = input_sort_by_str.strip().split("_")
+
+    descending = len(input_string) > 1 and input_string[1] == "desc"
+
+    if len(input_string) > 0 and (
+        input_string[0] == "created"
+        or input_string[0] == "modified"
+        or input_string[0] == "title"
+    ):
+        return input_string[0], descending
+    else:  # if invalid string is passed
+        return None, descending
```

Comment:
This is fine, but I don't think we really need to return the descending value here

Reply:
I take it back about this being fine-- I think we should return None, None here to eliminate the possibility of a bug that would sort by reverse search score